src/datepicker/Datepicker.vue

```diff
@@ -185,18 +195,34 @@ export default defineComponent({
     })
     const selectYear = (date: Date) => {
       pageDate.value = date
-      viewShown.value = 'month'
+
+      if (props.minimumView === 'year') {
+        viewShown.value = 'none'
+        emit('update:modelValue', date)
+      } else {
+        viewShown.value = 'month'
+      }
     }
     const selectMonth = (date: Date) => {
       pageDate.value = date
-      viewShown.value = 'day'
+
+      if (props.minimumView === 'month') {
+        viewShown.value = 'none'
+        emit('update:modelValue', date)
+      } else {
+        viewShown.value = 'day'
+      }
     }
     const selectDay = (date: Date) => {
       emit('update:modelValue', date)
 
       viewShown.value = 'none'
     }
 
+    const initialView = computed(() => {
+      return props.startingView === props.minimumView ? props.startingView : props.minimumView
```

Comment:
There seems to a bug here:

If the starting view is `year` and the minimum view is `day`, then the initial view will be `day` instead of `year`, despite it being a valid combination.

Reply:
Also, although I am yet to set it up properly, this line is not Prettier-compliant :)

Reply:
Sorry I missed your comment, and thank you for your review!
I fixed bugs and rebase the latest master branch, please review again.